Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2435,6 +2435,32 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
 
 // TODO: Consider making the return type be generic over `FloatingPoint` types
 // so that `self`'s scalar type can be any `Numeric` type.
+extension Tensor {
+    /// Returns `true` if given axis is in the range `[-rank, rank)`
```

Comment:
Doc comment fix: please remove "for the provided tensors". This comment doesn't make sense on an instance method of `Tensor`: the implicit given `Tensor` is `self`.
```suggestion
    /// Returns `true` if given axis is in the range `[-rank, rank)`.
```
Please remove "for the provided tensors" from other newly added doc comments please.